hermes-mock/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     compile group: 'com.jayway.awaitility', name: 'awaitility', version: '1.6.1'
     compile group: 'org.apache.avro', name: 'avro', version: versions.avro
     compile group: 'tech.allegro.schema.json2avro', name: 'converter', version: versions.json2avro
+    compile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.3.1'
```

Comment:
looks like we have junit 4 and 5 both on classpath?

Reply:
Jakub has suggested to add support for `Extensions` from JUnit 5.
Objections?

Reply:
I understand, it's fine.

Reply:
Thanks :)